amti/clis/preview.py

```diff
@@ -0,0 +1,100 @@
+"""CLI for running a simple web server to preview HIT"""
+
+import os
+import json
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from urllib.parse import urlparse
+
+import click
+import jinja2
+
+from amti import actions
+from amti import settings
+from amti import utils
+
+
+class MyServer(HTTPServer):
```

Comment:
Since there's only one server in this context, this class can probably just go by `Server`.